BaseTools/Scripts/GitHub.py

```diff
@@ -0,0 +1,160 @@
+## @file

```

Comment:
Would this file be better in .github directory since it isn't really useful for firmware development?

Reply:
another reason being we are not managing it's dependencies in basetools or edk2.  We are handling them inline in the workflow.

Reply:
Done